js/background.js

```diff
@@ -69,10 +69,70 @@
         return accountManager;
     };
 
+    function rollDice(text) {
+        var percentage = parseInt(text, 10);
+
+        if (isNaN(percentage)) {
+            return false;
+        }
+
+        var roll = _.random(1, 100);
+        if (roll <= percentage) {
+            return true;
+        }
+
+        return false;
+    }
+
+    var UPGRADE_VALUE = 'feb-2018-upgrade-dice';
+    var UPGRADE_FLAG = window.UPGRADE_FLAG = 'feb-2018-upgrade-alert';
+    var UPGRADE_URL = 'https://updates.signal.org/desktop/feb-2018-upgrade.txt';
+    var HALF_HOUR = 15 * 1000; // 30 * 60 * 1000;
```

Comment:
Should this be reverted to the value in the comments?

Reply:
Heh, yes. :)